build.sbt

```diff
@@ -52,6 +52,12 @@ lazy val commonSettings = Seq(
 
 ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8")
 
+ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
+
+// currently only publishing tags
+ThisBuild / githubWorkflowPublishTargetBranches :=
+  Seq(RefPredicate.StartsWith(Ref.Tag("v")))
+
```

Comment:
These settings seem like they could go into the org plugin at some point?  Particularly in the Great Schism?

Reply:
Absolutely unrelated to what's being done here, but what is "The Great Schism"?

Reply:
The break between the Roman Catholic and Eastern Orthodox Church in the 11th Century.

Alternatively, [this](https://github.com/http4s/http4s/discussions/4140).

Reply:
Thanks!  For both even